e2e/e2e_docker_test.go

```diff
@@ -17,20 +17,25 @@
 package e2e_test
 
 import (
+	"path/filepath"
 	"testing"
 
 	"github.com/GoogleContainerTools/kpt/pkg/test/runner"
 )
 
-func TestPipeline(t *testing.T) {
-	runPipelineTests(t, "../internal/pipeline/testdata/")
+func TestFnRender(t *testing.T) {
+	runTests(t, filepath.Join("..", "internal", "pipeline", "testdata"))
```

Comment:
Can you move this to e2e/ directory as well?

e2e/testdata/fn-eval
e2e/testdata/fn-render

Let's also standardize the package names. For render, it's currently:

`pkg-with-X`

For eval, it's

`eval-with`

Both qualifiers are unnecessary given the directory names:

e2e/testdata/fn-eval/simple-function
e2e/testdata/fn-render/simple-function

Reply:
I would prefer to have a separate PR for this. Will create one immediately after this one.

Reply:
I like this layout.

Reply:
#1660